src/ast.rs

```diff
@@ -7199,6 +7199,12 @@ where
 /// <special-name> ::= GR <object name> <seq-id> _    # Subsequent temporaries
 /// ```
 ///
+/// ### 5.1.4.6 Transaction-Safe Function
+///
+/// ```text
+/// <special-name> ::= GTt <encoding>
+/// ```
```

Comment:
You should add the non-safe version here too, right?

Reply:
Yup there should be.